### DISCUSSION TITLE
Organizationに参加している人にだけ見える情報領域

### DISCUSSION THREAD

**Discussion by n0bisuke** [increments/qiita-discussions#892] [要望・アイデア・不具合]
こんにちは

他の方が必要かというとたぶんあまり必要ないのですが... 要望書いてみます。

## 要望

Organizationに参加している人にだけ情報が見える領域があるとありがたいなと思っています。

connpassのイベントページの参加者だけに見せる限定URLのような使い方を想定しています。

> connpassのイベントページスクショ
> ![](https://i.gyazo.com/6c54bba5fe87275a193b95a9ddcfdf3b.png)

## 背景

特定の対象者にQiitaのオーガナイゼーションに入ってもらい、記事を書いてもらいたい という指示を出すときがあるのですが、対象者がオーガナイゼーションへの参加がそもそも出来てなく（メール通知に気づかない）、やり取りが都度発生してしまう問題があります。

オペレーションを組むときにオーガナイゼーションに入っていないと先の作業ができないように、オーガナイゼーションに入っている人にだけ限定URLを見せるような仕組みがあると良いなと思っています。

### 別アプローチ
オーガナイゼーションに気づかない問題はQiitaのホーム画面の通知に出てくるとありがたいなとも思っています。

ご検討の程よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@n0bisuke 
ご意見ありがとうございます！

着手まで時間がかかる可能性がありますが、Organization のアップデートを将来的に検討しております。
どのような形になるかはお約束できないのですが、下記のようなご意見は複数いただいているため

・Organizationメンバーだけが閲覧できる記事が欲しい
・Organizationの招待に気づきやすくして欲しい

n0bisukeさんにいただいたご意見も、合わせて検討したいと思います！

**Reply by n0bisuke** [reply]
コメントありがとうございます！

正座待機してます。
